python/doc/examples/calibration/bayesian_calibration/plot_ackley_distribution.py

```diff
@@ -181,8 +181,9 @@ def _exec(self, X):
 legends = [format_float_scientific(float(v), precision=1) for v in legends[:-1]]
 contour.setLegends(legends)
 graph.add(contour)
-view = View(graph, legend_kw={"bbox_to_anchor": (1, 1), "loc": "upper left"})
-view.getFigure().tight_layout()
+graph.setLegendCorner([1, 1])
```

Comment:
The original code was wrong: the input argument is a `float`, not an `int`.
```suggestion
graph.setLegendCorner([1.0, 1.0])
```